### DISCUSSION TITLE
Descriptive error message when Theme context is not found

### DISCUSSION THREAD

**Discussion by alecnicolas** [twilio-labs/paste#597] [General] [answered]
Hello, thanks for working on Paste!

When using the Tooltip component and running the build on Jenkins, I ran into this error:

![image](https://user-images.githubusercontent.com/29192545/88117033-e65e4400-cb6e-11ea-97df-54c764c9a317.png)

After looking into this error, it would appear that it is due to the tests not having the theme context and so fontSize70 can't be read from the referenced theme. An error message that points that out to the user could help give more clarity when debugging something similar.

**Comment by serifluous** [comment]
Thanks, @alecnicolas! That's a great point. I'll bring this up with the team and see what we can do.

**Comment by SiTaggart** [comment, marked as answer]
Hi @alecnicolas, it's certainly not ideal and it trips folks up alot of the time. We've called it out in the getting started docs https://paste.twilio.design/getting-started/engineering#jest-testing but adding some kind of check might be beneficial in the long run.

It's a little bit tricky to do right now as there are a few places we'd need to check based on certain component implementations. There isn't currently a central place where we could check, for any given component, are you in a theme provider. In the near future once all our components are just using `Box` under the hood, we could just do a single check inside of `Box`. Check if there is a `theme`, if not, console.error or something.

It's definitely something we could consider in the future. Thanks for the suggestion 🎉
